### PR TITLE
Update Fastly set up instructions and SSL/TLS certificate information

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

- Updated Site launch and launch checklist for Platform TLS and DNS configuration updates for production
- Updated Fastly set up topic with instructions for SSL/TLS certificate provisioning and DNS configuration updates
- Added links to Fastly docs for basic authentication and edge module sections in Customi
- Fixed format and links in Fastly set up topic
- Updated Fastly overview topic
   - Updated security section to add bullet points for origin cloaking and SSL/TLS certificates
   -  Removed duplicated content covered in custom cache configuration sections
   - Changed inline links to reference links in "link definitions" section
   remove duplicated content covered in custom cache configuration sections
- New _Custom cache configuration_ topic
   - Moved custom cache configuration sections from Fastly set up topic
   - Added instructions and screen cap for Managing domains on Starter
projects
   - Add instructions and screen cap for Enabling maintenance mode
during development

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/cdn/cloud-fastly.html
- https://devdocs.magento.com/cloud/cdn/configure-fastly-customize-cache.html
- https://devdocs.magento.com/cloud/cdn/configure-fastly.html
- https://devdocs.magento.com/cloud/live/live.html
- https://devdocs.magento.com/cloud/live/site-launch-checklist.html
- https://devdocs.magento.com/cloud/live/stage-prod-migrate-prereq.html
- https://devdocs.magento.com/cloud/cdn/cloud-vcl-custom-snippets.html
- https://devdocs.magento.com/cloud/cdn/fastly-image-optimization.html
- https://devdocs.magento.com/cloud/cdn/src/cloud/cdn/trouble-fastly.html
- https://devdocs.magento.com/cloud/live/stage-prod-migrate-prereq.html
- https://devdocs.magento.com/cloud/requirements/cloud-requirements.html

whatsnew

- Updated Fastly set up and Site launch documentation in the Cloud Guide to reflect change from shared certificates for Fastly for each project to have a unique Domain-Validated SSL/TLS certificate issued by Let's Encrypt. Also, updated DNS configuration instructions for adding domains, CNAME, and A records.

- Moved documentation for Fastly cache customization options to a new Custom cache configuration topic. Added instructions for managing domains and enabling maintenance mode using the Fastly CDN module for Magento 2. See Custom cache configuration.

### CODE REVIEW

**Comment by hguthrie** [comment]
It's the big one! I'm on it...

File: src/cloud/cdn/configure-fastly-customize-cache.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+1. In the *Fastly Configuration* section, expand **Advanced**.
+
+1. Set the **Admin path timeout** value in seconds. This value cannot be more than 10 minutes (600 seconds).
+
+1. Click **Save Config** at the top of the page.
+
+1. After the page reloads, click **Upload VCL to Fastly** in the *Fastly Configuration* section.
+
+Fastly gets the Magento Admin path for generating the VCL file from the `app/etc/env.php` configuration file.
```
Comment (hguthrie) [inline src/cloud/cdn/configure-fastly-customize-cache.md:43]: "gets" is a little vague.

File: src/cloud/cdn/configure-fastly-customize-cache.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+-  **Soft purge**–Sets changed content to stale and purges according to the stale timing. In combination with the stale timings your customers will be served stale content very fast while Fastly is updating the content in the background.
+
+![Configure purge options]({{ site.baseurl }}/common/images/cloud/cloud_fastly-purgeoptions.png){:width="650px"}
+
+{:.procedure}
+To configure Fastly purge options:
+
+1. In the *Fastly Configuration* section, expand **Advanced**.
+
+1. All purge options display. Select "Yes" per purge option to enable automatic purging. Select "No" to disable automatic purging, allowing you to manually purge caches through the Cache Management page.
```
Comment (hguthrie) [inline src/cloud/cdn/configure-fastly-customize-cache.md:63]: "per purge option" I wonder if the use of **per** will be confusing to some readers. Maybe use "for each" or something similar?

File: src/cloud/cdn/trouble-fastly.md [hunk 3]
```diff
@@ -59,7 +59,8 @@ Use the following list to identify and troubleshoot issues related to the Fastly
 If Fastly returns 503 timeout errors, check the error logs and the 503 error page to identify the root cause.
 
 {:.bs-callout-info}
-If the timeout occurs when running bulk operations, you can [extend the Fastly timeout for the Magento Admin UI]({{ site.baseurl }}/cloud/cdn/configure-fastly.html#bulkaction).
+If the timeout occurs when running bulk operations, you can [extend the Fastly timeout for the Magento Admin UI]
+({{site.baseurl}}/cloud/cdn/configure-fastly-customize-cache.html#extend-fastly-timeout).
```
Comment (hguthrie) [inline src/cloud/cdn/trouble-fastly.md:63]: Line break in the middle of a URL. Note is not showing properly.

**Review by hguthrie — CHANGES_REQUESTED** [review]
I have to request a change for one broken link.
I did see a couple of other things that might be confusing, but they are not show-stoppers.
Otherwise, good to go!

File: src/cloud/cdn/configure-fastly-customize-cache.md (hunk 1 above)
Comment (meker12) [inline src/cloud/cdn/configure-fastly-customize-cache.md:43]:
```suggestion
Fastly retrieves the Magento Admin path for generating the VCL file from the `app/etc/env.php` configuration file.
```

File: src/cloud/cdn/trouble-fastly.md (hunk 3 above)
Comment (meker12) [inline src/cloud/cdn/trouble-fastly.md:63]:
```suggestion
If the timeout occurs when running bulk operations, you can [extend the Fastly timeout for the Magento Admin UI]({{site.baseurl}}/cloud/cdn/configure-fastly-customize-cache.html#extend-fastly-timeout).
```

**Review by hguthrie — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Pull request merged**